lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -117,12 +120,19 @@ module.exports = function(
    * @function
    * @public
    *
+   * @param {String} method - how the modal is closed, used for logging
```

Comment:
This looks weird, I think we should avoid thinking how the modal was closed here. We should send a generic "drive selector closed" event, while emitting all the other low level events from the modal service instead (in a generic way so that it applies to all modals).